nemo/utils/callbacks/nemo_model_checkpoint.py

```diff
@@ -208,6 +208,8 @@ def on_save_checkpoint(self, trainer, pl_module, checkpoint):
         app_state = AppState()
         if app_state.model_parallel_size is not None and app_state.model_parallel_size > 1:
             logging.warning(f'always_save_nemo will slow down training for model_parallel > 1.')
+        if app_state.context_parallel_size is not None and app_state.context_parallel_size > 1:
+            logging.warning(f'always_save_nemo will slow down training for context_parallel > 1.')
```

Comment:
trying to understand this, why will it slow down training? does it take longer to checkpoint with both cp>1 and `always_save_nemo`? is it still a slowdown if we are using async checkpointing?

Reply:
It is actually `always_save_nemo` is not supported with any model parallel including CP

Reply:
Ok, then 'slowdown' is misleading. But then I think the 'error' in exp_manager is sufficient, do we need this warning as well?

Reply:
I think we can remove this, only use the exp_manager one then.